### PR TITLE
Add support for running multiple schema updates from one migration file

### PR DESCRIPTION
**Pull request by techgerm**
* Issue: https://github.com/maroux/python-spanner-orm/issues/3
* Tested with live Spanner instance and works great

P.S.
* We could use a new release/publish after this PR is merged. Your time is much appreciated 🙌

### CODE REVIEW

File: spanner_orm/admin/migration.skel [hunk 1]
```diff
@@ -16,3 +16,12 @@ def upgrade() -> spanner_orm.NoUpdate:
 # Returns a SchemaUpdate object that tells how to roll back the changes
 def downgrade() -> spanner_orm.NoUpdate:
   return spanner_orm.NoUpdate()
+
+# Note: upgrade/downgrade can also return a list of SchemaUpdates
+#       which will each be executed in sequential order
```
Comment (maroux) [inline spanner_orm/admin/migration.skel]: I'd add a note here that downgrade should reverse the list of operations.

**Review by maroux — COMMENTED** [review]
Niceee! 💯

File: spanner_orm/admin/migration.skel (hunk 1 above)
Comment (techgerm) [reply spanner_orm/admin/migration.skel]: good call 👌

**Review by maroux — APPROVED** [review]

**Review by maroux — APPROVED** [review]

**Comment by techgerm** [comment]
@maroux do you think we could get any estimate on the next release date? 😁

**Comment by maroux** [comment]
> @maroux do you think we could get any estimate on the next release date? 😁

Published.

**Comment by techgerm** [comment]
Thanks my friend! 🙌

**Pull request merged**